frontend/src/components/date_time_picker/index.tsx

```diff
@@ -0,0 +1,86 @@
+import 'react-day-picker/dist/style.css';
+import React, { useEffect, useState } from 'react';
+
+import { DayPicker, SelectSingleEventHandler } from 'react-day-picker';
+import classnames from 'classnames/bind';
+
+import Popover from 'src/components/popover';
+import Button from 'src/components/button';
+
+const cx = classnames.bind(require('./stylesheet'));
+
+interface DateTimePickerProps {
+	onSelectedDate: (date: Date) => void;
+	selected?: Date | undefined;
+}
+
+interface PickerProps extends DateTimePickerProps {
+	onButtonClick: () => void;
+}
+
+const Picker: React.FC<PickerProps> = ({ onButtonClick, onSelectedDate, selected }) => {
+	const [date, setDate] = useState<Date>(selected ?? new Date());
+
+	useEffect(() => {
+		onSelectedDate(date)
+	}, [date, onSelectedDate])
+
+	const onChangeDate: SelectSingleEventHandler = (newDate) => {
+		setDate(new Date(newDate as Date))
+	}
+
+	const onChangeHour: React.ChangeEventHandler<HTMLInputElement> = (event) => {
+		const [hours, minutes] = event.target.value.split(":").map(value => Number(value))
+		const newDate = date
+
+		newDate.setHours(hours, minutes)
+
+		setDate(new Date(newDate))
```

Comment:
I believe you could replace this with a datefns function:

```suggestion
		setDate(setHours(date, hours))
```

Reply:
I'll check it out, but probably I'll have to use setHours and setMinutes, whereas with the setHours function from the date object I can do it all in one call only